.circleci/config.yml

```diff
@@ -66,6 +66,11 @@ jobs:
       - image: gcc:5.5
     steps:
       - checkout
+      - run:
+          name: checkout embedded-sht
+          command: |
+              git clone git@github.com:Sensirion/embedded-sht.git ../embedded-sht
+              cd ../embedded-sht && make sht-common/sht_git_version.c && cd -
```

Comment:
A trick I learned from `shellcheck`: run in subshell to avoid having to `cd -`. Also, run the new `make prepare` instead of the target file that hopefully won't be renamed:

`(cd ../embedded-sht && make preapre)`